Development-Guide/AsciiDoc-Guide/README.adoc

```diff
@@ -1,4 +1,15 @@
-= AsciiDoc
+= AsciiDoc Guide
+
+Any plain text documentation should be done in AsciiDoctor syntax in the Solar Car Team.
```

Comment:
I think better wording would be `Any plain text documentation on the Solar Car Team should be done in AsciiDoctor syntax`